extensions/parquet/table/src/main/java/io/deephaven/parquet/table/pagestore/topage/ToLongPage.java

```diff
@@ -42,7 +58,13 @@ public final ChunkType getChunkType() {
 
     @Override
     @NotNull
-    public final Object nullValue() {
+    public Object nullValue() {
```

Comment:
Why did we make this non `final`? I prefer to keep methods final until we have a reason to override them.